src/main/java/com/iota/iri/network/TransactionRequester.java

```diff
@@ -20,17 +20,17 @@
     private final MessageQ messageQ;
     private final Set<Hash> milestoneTransactionsToRequest = new LinkedHashSet<>();
     private final Set<Hash> transactionsToRequest = new LinkedHashSet<>();
+
+    public static final int MAX_TX_REQ_QUEUE_SIZE = 10000;
```

Comment:
configurable?

Reply:
this class don't have access to configuration.
as configuration will be redone shortly, it's not worth changing the interface imo